submodules/vmbox/vmbox.js

```diff
@@ -352,7 +352,7 @@ define(function(require) {
 					return re.test(email);
 				},
 				getRecipients = function() {
-					var list = $('#recipients_list', vmbox_html).val().replace(/\s+/g, '').split(',');
+					var list = $('#recipients_list', vmbox_html).val().replace(/,\s+/g,",").split(/[\n,\s+]/);
```

Comment:
I think it would be better to add all possible separation characters to the replace regex, and split only by commas. Also, according to the ticket, there should be an error message if the e-mails are invalid.